apps/dispatch/src/app/components/create-simulation-project/create-simulation-project/create-simulation-project.component.ts

```diff
@@ -45,6 +45,37 @@ import { validateValue } from '@biosimulations/datamodel/utils';
 import { ConfigService } from '@biosimulations/config/angular';
 import { HtmlSnackBarComponent } from '@biosimulations/shared/ui';
 import { FileInput } from 'ngx-material-file-input';
+import {
```

Comment:
Looks like these should be moved to a common data model repo, since this is being imported from NestJS specific code
A better idea might be to generate clients for angular as well

Reply:
I haven't dealt with that yet.

Reply:
yup, just thinking out loud re if we want a separate library for angular or just datamodel